src/TRoomDB.h

```diff
@@ -113,8 +114,10 @@ class TRoomDB
     QMap<int, QString> areaNamesMap;
     TMap* mpMap;
     QSet<int>* mpTempRoomDeletionSet; // Used during bulk room deletion
+    bool mBulkDeletionMode = false; // Flag to prevent expensive individual cleanup during bulk destruction
```

Comment:
Putting the comment *Before* the member means the comment now shows up in some situations in the Qt Creator IDE when you hover over the member in other places...! :astonished:
```suggestion
    // Flag to prevent expensive individual cleanup during bulk destruction
    bool mBulkDeletionMode = false;
```